src/flownet/templates/webviz_ahm_config.yml.jinja2

```
@@ -17,7 +17,7 @@ pages:
         {%- for iteration in [iterations[0], iterations[-1]] %}
          - iter-{{ iteration }}
         {%- endfor %}
-        observation_file: ./observations.yamlobs
+        observation_file: ./observations_complete.yamlobs
```

These observations will be used in the misfit calculation and that should only be calculated on the training observations.

```suggestion
        observation_file: ./observations_training.yamlobs
```